git-cliff-core/src/repo.rs

```diff
@@ -976,4 +1065,137 @@ mod test {
             assert!(!retain, "exclude: **/*.txt");
         }
     }
+
+    /// Test the semantic version comparison function
+    #[test]
+    fn test_semantic_version_compare() {
+        use std::cmp::Ordering;
+
+        // Test basic semantic version comparison
+        assert_eq!(
+            semantic_version_compare("v0.9.0", "v0.10.0"),
+            Ordering::Less
+        );
+        assert_eq!(
+            semantic_version_compare("v0.10.0", "v0.9.0"),
+            Ordering::Greater
+        );
+        assert_eq!(
+            semantic_version_compare("v1.0.0", "v1.0.0"),
+            Ordering::Equal
+        );
+
+        // Test without 'v' prefix
+        assert_eq!(semantic_version_compare("0.9.0", "0.10.0"), Ordering::Less);
+        assert_eq!(semantic_version_compare("1.2.3", "1.2.4"), Ordering::Less);
+
+        // Test with pre-release versions
+        assert_eq!(
+            semantic_version_compare("v1.0.0-alpha.1", "v1.0.0-alpha.2"),
+            Ordering::Less
+        );
+
+        // Test pre-release vs release (lines 574-575)
```

Comment:
Pointing out to lines that might change in the future does not make sense, can you remove them and reference to the function instead?